pkg/daemon/daemon.go

```diff
@@ -30,38 +26,22 @@ func New(c Config) *Daemon {
 		})
 	}
 
+	var met metric.Meter
+	{
+		met = recorder.NewMeter(recorder.MeterConfig{
+			Env: c.Env.Environment,
+		})
+	}
+
 	log.Log(
 		"level", "info",
 		"message", "daemon is starting",
 		"environment", c.Env.Environment,
 	)
 
-	var reg registry.Interface
-	{
-		reg = registry.New(registry.Config{
-			Log: log,
-			Met: mustCreateOpenTelemetryMeterInterface(c),
-		})
-	}
```

Comment:
Earlier I wasn't sure how to inject the registry. In this iteration I figured out how I would like to register metrics and that then provided insight into the more desirable dependency graph.

* Everything receives the OTEL meter.
* Every implementation of a server handler and worker handler uses the meter and defines their own metrics whitelist explicitly within their own private context.